### ISSUE TITLE
Repo is always added

### ISSUE THREAD

**Issue by throwaway99** [dj-wasabi/puppet-zabbix#148]
In zabbix::repo the if ($manage_repo) { does not seem to do anything.  If you define zabbix::repo as false the repo is still added.

This is applying the class zabbix. If you only apply zabbix::agent the repo is not added.

**Comment by dj-wasabi** [comment]
Hi @throwaway99 

Could you please show me your site manifest?

When I have this:
```
class { 'zabbix::agent':
  server      => 'zabbix.example.com',
  manage_repo => false,
}
```
it works like it should. No yum repository file is created.

**Comment by throwaway99** [comment]
Hi,

It's a Zabbix server and it's defined in Hiera as:-

classes:
  - 'zabbix'

# Zabbix configuration
zabbix::database_type: 'mysql'
zabbix::manage_firewall: false
zabbix::manage_repo: false
zabbix::manage_vhost: false

The example you give works fine as servers with 

classes:
  - 'zabbix::agent'

do not have the repository applied but where I apply 

classes: 
  - 'zabbix'

it does create the repository.  Hope this helps.

**Comment by dj-wasabi** [comment]
I pushed an fix to the repo and created an release 2.1.1. Can you verify that it works as expected?

**Comment by throwaway99** [comment]
Hi. Yes works as expected.  Thanks for this and the module!
